canonical-web-and-design/jaas.ai
# Use image_template module on homepage

## Done

Added image_template module on to the majority of images the homepage

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check all images load on the site homepage
